### PR TITLE
add "types" to service data model

### PR DESCRIPTION
**Pull request by allenjprice**

### CODE REVIEW

File: src/clj/odin/graphql/resolvers/service.clj [hunk 1]
```diff
@@ -26,6 +27,15 @@
   (keyword "service.billed" (name billed)))
 
 
+(defn svc-type
+  [_ _ service]
+  (-> (service/type service) name keyword))
+
```
Comment (jalehman) [inline src/clj/odin/graphql/resolvers/service.clj]: newline

File: src/clj/odin/graphql/resolvers/service.clj [hunk 2]
```diff
@@ -77,6 +88,7 @@
   [params]
   (tb/transform-when-key-exists params
     {:billed   #(keyword "service.billed" (name %))
+     :type     #(keyword "service.type" (name %))
```
Comment (jalehman) [inline src/clj/odin/graphql/resolvers/service.clj]: Can you use `make-type-key` here?

Comment (allenjprice) [reply src/clj/odin/graphql/resolvers/service.clj]: yep. same for the previous line, with `make-billed-key`.

**Comment by allenjprice** [comment]
@jalehman changes incorporated 🎉

**Pull request merged**